### PR TITLE
Axes options and Form Alterations

### PR DESCRIPTION
**Pull request by lilyclements**
Changed some small things in the main form - added in the next dialog (Use Model) although haven't started this yet. Just wanted to add it in right away to avoid conflicts!
Also, added in the 'comment' for changing values
And for axes, it now has 'interval' as the default

### CODE REVIEW

File: instat/frmEditor.vb [hunk 1]
```diff
@@ -80,8 +80,8 @@ Public Class frmEditor
         clsInsertColumns.AddParameter("num_cols", grdCurrSheet.SelectionRange.Cols)
         clsInsertColumns.AddParameter("before", "TRUE")
         'TODO This should be an option in dialog
-        clsInsertColumns.AddParameter("col_name", Chr(34) & "X" & Chr(34))
-        clsInsertColumns.AddParameter("use_col_name_as_prefix", "TRUE")
+        clsInsertColumns.AddParameter("col_names", Chr(34) & "X" & Chr(34))
+        clsInsertColumns.AddParameter("use_col_names_as_prefix", "TRUE")
```
Comment (dannyparsons) [inline instat/frmEditor.vb]: Are you sure these are the right parameter names?

Comment (dannyparsons) [inline instat/frmEditor.vb]: These are Instat object methods so the parameter names are in the R code.

Comment (lilyclements) [inline instat/frmEditor.vb]: Sorry, that was an accidental change when I was getting an error message about something else so  I was experimenting. I thought I undid it! Sorry!

Comment (dannyparsons) [inline instat/frmEditor.vb]: No problem. Can you undo these changes?

Comment (lilyclements) [inline instat/frmEditor.vb]: Done!
